README.md

```diff
@@ -1,10 +1,25 @@
 # APIToolKit Symfony SDK
-A PHP/Symfony SDK Wrapper for APIToolkit. It monitors incoming traffic, gathers the requests and sends the request to the apitoolkit. The SDK simply registers and event listener and consumes this information via the events.
+
+A PHP/Symfony SDK Wrapper for APIToolkit. It monitors incoming traffic, gathers the requests and sends the request to
+the apitoolkit. The SDK simply registers and event listener and consumes this information via the events.
 
 ## Installation
+
 Run the following command to install the package:
 
 composer require apitoolkit/apitoolkit-symfony
+
+add the listener to your service.yaml
+
+```yaml
+services:
+
+  APIToolkit\EventSubscriber\APIToolkitService:
+    arguments:
+      - '%env(APITOOLKIT_KEY)%'
+      - 'https://app.apitoolkit.io' 
```

Comment:
Ideally, this field should be optional. Is there a way to configure this? I expect it to only be used for debugging purposes/
```suggestion
      - 'https://app.apitoolkit.io' 
```